package.json

```diff
@@ -228,7 +228,7 @@
   "private": true,
   "dependencies": {
     "@department-of-veterans-affairs/formation": "^6.9.0",
-    "@department-of-veterans-affairs/formation-react": "^5.1.0",
+    "@department-of-veterans-affairs/formation-react": "^5.1.1",
```

Comment:
I think it was harmless, but I'm afraid I unknowingly had a change in my working directory of the design system to the promo banner when I published this. Can we hold until this is merged and version 5.2.0 published? https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/207

Reply:
Totally! 🙂